contrib/go-pg/pg_go.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pg
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-pg/pg/v10"
```

Comment:
This goes in a separate group at the end, as described in https://github.com/DataDog/dd-trace-go/pull/686#discussion_r448229477. Please check other files or integrations for more clarity.